packages/headless/src/features/facets/range-facets/numeric-facet-set/numeric-facet-controller-actions.ts

```diff
@@ -28,10 +33,7 @@ export const executeToggleNumericFacetSelect = createAsyncThunk<
 >(
   executeToggleNumericFacetSelectType,
   (payload, {dispatch, extra: {validatePayload}}) => {
-    validatePayload(payload, {
-      facetId: facetIdDefinition,
-      selection: new RecordValue({values: numericFacetValueDefinition}),
-    });
+    validatePayload(payload, definition);
```

Comment:
General question on a pattern I saw in this PR, why is `validatePayload` passed as a thunk argument instead of importing it?

Reply:
I described it a bit here: https://github.com/coveo/ui-kit/pull/310#issuecomment-738814156